### PR TITLE
Newsletter 10

### PR DESCRIPTION
**Pull request by ozkriff**
Preview (6cb7dfb): https://rust-gamedev-preview.netlify.com/posts/newsletter-010

Everyone is invited to review!

:timer_clock:  Hope to merge this tomorrow +- at 17:00 UTC (in ~20 hrs from now) if there're no objections.

_Part of #139 ("Newsletter 10: Coordination/Tracking")_

### CODE REVIEW

File: content/posts/newsletter-010/index.md [hunk 1]
```diff
@@ -201,7 +200,7 @@ with highlights also published on [YouTube][sailing-youtube].
 
 [Garden][garden] is an upcoming game centered around growing realistic plants.
 
-[May][garden-devlog] devlog were posted.
+[May][garden-devlog] devlog wes posted.
```
Comment (azriel91) [inline content/posts/newsletter-010/index.md:203]:
```suggestion
[May][garden-devlog] devlog was posted.
```

Comment (ozkriff) [reply content/posts/newsletter-010/index.md]: fixed, thanks

**Comment by Wodann** [comment]
I could offer a suggestion, as this PR didn't edit the particular line, but this markdown is not correct: https://github.com/rust-gamedev/rust-gamedev.github.io/pull/169/files#diff-87a6f90e4882f9efb876defd3a0f053dR571

image

**Comment by kvark** [comment]
@ozkriff thank you for all the amazing work on this newsletter!

**Pull request merged**